pkg/pillar/cmd/zedagent/zedagent.go

```diff
@@ -235,10 +235,68 @@ var logger *logrus.Logger
 var log *base.LogObject
 var zedcloudCtx *zedcloud.ZedCloudContext
 
+type infoDest uint
```

Comment:
What about `publishDest` instead to make it more clear and cover metrics as well?

Reply:
And maybe also with `s` or `Set` suffix to make it clear that a value of this type is essentially a set, built using bit flags.

Reply:
Bitset?

Reply:
The suffix, infoDestBitset

Reply:
+1 to use a different name to cover metrics as well. publishDest or apiDest are names to consider.